build/run.js

```diff
@@ -70,7 +72,17 @@ function command(docs) {
     return {docs}
 }
 
-
+function run_project_manager() {
+    const bin_path = paths.get_project_manager_path(paths.dist.bin)
+    console.log('Starting language server from ' + bin_path)
+    child_process.execFile(bin_path, [], (error, stdout, stderr) => {
+        console.error(stderr)
+        if (error) {
+            throw error
+        }
+        console.log(stdout)
+    })
```

Comment:
is this reporting errors / logs interactively, while running the process?

Reply:
I don't think it is. From the docs it sounds like this is only called on the end of the process.

Reply:
Call note: ide/build/cmd.js